### ISSUE TITLE
Laggy scaling animation in Firefox - affects all versions

### ISSUE THREAD

**Issue by drawcard** [Glitchbone/jquery-fatNav#6]
If you visit http://glitchbone.github.io/jquery-fatNav/ in Firefox and use FatNav on a Macbook Pro 15" Retina display (1920x1200 / "More Space" resolution setting), the scaling & fading transition animation is very laggy / slow. However if you scale the browser window down to say 1/4 screen size, the speed is OK.

**Comment by Glitchbone** [comment]
Hi,
I think the issue comes from the Retina display as I tested the plugin on Firefox on my non-Retina iMac and the transition runs smoothly

**Comment by drawcard** [comment]
Yeah, it's definitely to do with Retina displays. It seems to run OK on my other 1920x1080 px monitor.
